README.md

```diff
@@ -103,33 +103,65 @@ EOF
     defer_save = { "InsertLeave", "TextChanged" }, -- vim events that trigger a deferred save (saves after `debounce_delay`)
     cancel_defered_save = { "InsertEnter" }, -- vim events that cancel a pending deferred save
   },
-  -- function that determines whether to save the current buffer or not
+  -- function that takes the buffer handle and determines whether to save the current buffer or not
   -- return true: if buffer is ok to be saved
   -- return false: if it's not ok to be saved
-  condition = function(buf)
-    local fn = vim.fn
-    local utils = require("auto-save.utils.data")
-
-    if
-      fn.getbufvar(buf, "&modifiable") == 1 and
-      utils.not_in(fn.getbufvar(buf, "&filetype"), {}) then
-      return true -- met condition(s), can save
-    end
-    return false -- can't save
-  end,
+  -- if set to `nil` then no specific condition is applied
+  condition = nil,
   write_all_buffers = false, -- write all buffers when the current one meets `condition`
   debounce_delay = 1000, -- delay after which a pending save is executed
   callbacks = { -- functions to be executed at different intervals
     enabling = nil, -- ran when enabling auto-save
     disabling = nil, -- ran when disabling auto-save
-    before_asserting_save = nil, -- ran before checking `condition`
     before_saving = nil, -- ran before doing the actual save
     after_saving = nil -- ran after doing the actual save
   }
 }
 ```
 
-Additionally you may want to set up a key mapping to toggle auto-save:
+#### Condition
+The condition field of the configuration allows the user to exclude **auto-save** from saving specific buffers.
+There is a helper functions provided for not saving specified file types:
```

Comment:
Wording: "Here's an example using a helper function from `auto-save.utils.data` that disables `auto-save` for specified file types:"

I know this is nitpicky but for a second I was a bit confused because I thought that the "helper functions" referred to the examples below and those aren't defined anywhere besides readme.

Reply:
No, this is totally fine :)
Fixed it, feel free to merge.